packages/components/package.json

```diff
@@ -145,19 +144,13 @@
         "@semantic-release/git",
         {
           "message": "chore(release/components): ${nextRelease.version} [skip actions]\n\n${nextRelease.notes}",
-          "assets": [
-            "CHANGELOG.md",
-            "package.json"
-          ]
-        }
-      ],
-      [
-        "@semantic-release/github",
-        {
-          "successComment": false
+          "assets": ["CHANGELOG.md", "package.json"]
         }
       ]
-    ]
+    ],
+    "monorepo": {
+      "dependencies": ["packages/vue", "packages/react", "packages/angular"]
```

Comment:
Really like the ease of configuration here!

Reply:
Thanks!